dashboard/static/styles/elements.styl

```diff
@@ -3,3 +3,11 @@
 a {
     color: $blue;
 }
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: "Fira Sans";
+}
+
+h1 {
+    font-size: 2.5rem;
+}
```

Comment:
If I'm not mistaken, it looks like we can remove this now. The site title is the only `<h1>` and the size from the `.top-bar-title h1` rule is used instead.